heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/executor/SinkExecutorTest.java

```diff
@@ -38,9 +41,12 @@
 import com.twitter.heron.spi.metricsmgr.sink.IMetricsSink;
 import com.twitter.heron.spi.metricsmgr.sink.SinkContext;
 
+import static org.junit.Assert.fail;
+
 public class SinkExecutorTest {
-  private static final int FLUSH_INTERVAL_MS = 100;
+  private static final long FLUSH_INTERVAL_MS = 100;
   private static final int N = 100;
```

Comment:
It took all I had to not update that field :) because I didn't want to clutter up the diff. If everything else looks ok I can make that change.